src/application.ts

```diff
@@ -193,7 +193,7 @@ export default class Application {
     Object.keys(data.relationships).forEach(relationshipName => {
       const relationships = this.serializeRelationship((data.relationships[
         relationshipName
-      ] as unknown) as Resource | Resource[]);
+      ] as unknown) as Resource | Resource[], schemaRelationships[relationshipName].type().schema.primaryKeyName);
```

Comment:
@spersico Can we create some variables to make these lines a bit more... readable? 🙏